src/components/ArtifactSigner/artifact-signer.tsx

```diff
@@ -0,0 +1,270 @@
+      </ProgressStepper>
+    </>
+  );
+};
+
+type DownloadPullSecretStepProps = {
+  onDownloadSecret: () => void;
+  canProceed: boolean;
+  onPrevious: () => void;
+  onNext: () => void;
+};
+const DownloadPullSecretStep = ({
+  canProceed,
+  onDownloadSecret,
+  onNext,
+  onPrevious,
+}: DownloadPullSecretStepProps) => {
+  return (
+    <div
+      style={{
+        marginTop: '1rem',
+      }}
+    >
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Download the pull secret needed for your deployment
+        </Text>
+        <div
+          style={{
+            flex: 1,
+            flexDirection: 'column',
+            display: 'inline-flex',
+          }}
+        >
+          <Button variant="primary" onClick={onDownloadSecret}>
+            Download Pull Secret
+          </Button>
+          <div
+            style={{
+              paddingTop: '1rem',
+            }}
+          >
+            <Button variant="link" isSmall onClick={onPrevious}>
+              <ArrowLeftIcon /> Previous
+            </Button>
+            <Button
+              variant="secondary"
+              isSmall
+              isDisabled={!canProceed}
+              onClick={onNext}
+            >
+              Next
+            </Button>
+          </div>
+        </div>
+      </TextContent>
+    </div>
+  );
+};
+
+type InstallArtifactSignerStepProps = {
+  onPrevious: () => void;
+  onNext: () => void;
+  onDocumentationClick: () => void;
+};
+const InstallArtifactSignerStep = ({
+  onNext,
+  onPrevious,
+  onDocumentationClick,
+}: InstallArtifactSignerStepProps) => {
+  return (
+    <div>
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Install Trusted Artifact Signer on an OpenShift cluster by following
+          the instructions in the documentation
+        </Text>
+        <div
+          style={{
+            flex: 1,
+            flexDirection: 'column',
+            display: 'inline-flex',
+            justifyItems: 'center',
+          }}
+        >
+          <Button
+            variant="link"
+            onClick={onDocumentationClick}
+            iconPosition="right"
+            icon={<ExternalLinkSquareAltIcon />}
+          >
+            Trusted Artifact Signer Documentation
+          </Button>
+          <div
+            style={{
+              paddingTop: '1rem',
+            }}
+          >
+            <Button variant="link" isSmall onClick={onPrevious}>
+              <ArrowLeftIcon /> Previous
+            </Button>
+            <Button variant="secondary" isSmall onClick={onNext}>
+              Finish
+            </Button>
+          </div>
+        </div>
+      </TextContent>
+    </div>
+  );
+};
+
+type SuccessStepProps = {
+  onPrevious: () => void;
+};
+const SuccessStep = ({ onPrevious }: SuccessStepProps) => {
+  return (
+    <div>
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Trusted Artifact Signer is installed and ready to use
```

Comment:
Do we really know this is the case? There is no feedback from the installation steps is there? Maybe it's better to say something like "If you have followed the documentation for installation, Trusted Artifact Signer is installed and ready to use".

Reply:
Yeah, let's just remove the stepper and place the documentation + pull secret to be on the same screen.